src/api/controllers/ActionUserController.js

```diff
@@ -1,17 +1,88 @@
-module.exports = {
-    addAttendant: (req, res) => {
+const Action = require("../models/ActionModel");
+const User = require("../models/UserModel");
+const mongo = require("mongoose");
+
+mongo.set("useFindAndModify", false);
+
+const updateAttendants = (req, res, add) => {
+  const userId = req.userData.userId;
+  const actionId = req.params.action_id;
+  var actionQuery = {};
+  var userQuery = {};
+
+  if (add) {
+    actionQuery["$addToSet"] = { attendees: userId };
+    userQuery["$addToSet"] = { actionsAttended: actionId };
+  } else {
+    actionQuery["$pull"] = { actionsAttended: actionId };
```

Comment:
This is wrong and is causing the action document to not be updated.
```suggestion
    actionQuery["$pull"] = { attendees: actionId };
```